crates/astria-proto/src/native/sequencer/v1alpha1.rs

```diff
@@ -209,6 +213,8 @@ impl SignedTransaction {
 pub struct UnsignedTransaction {
     pub nonce: u32,
     pub actions: Vec<Action>,
+    /// asset to use for fee payment.
+    pub fee_asset_id: asset::Id,
```

Comment:
See above, not opposed to types, just wondering if leaving this as a bare `[u8; 32]` would be fine.